apps/voting-disputable/test/helpers/deployer.js

```diff
@@ -0,0 +1,147 @@
+const { pct } = require('./voting')()
```

Comment:
So I know you really like deployer classes, but is there a reason to prefer this over a simplified version with just functions that don't encapsulate any state?

I'm definitely of the mindset that the less magic and abstraction there are in tests, especially in terms of hidden state, the better :).

Reply:
I found this one relatively simple compared to the agreement app, for example, so with a bunch of functions, it was straightforward to test. But I agree we must find a rationale and follow the same approach always.